src/main/java/com/example/flight/FlightApplication.java

```diff
@@ -1,13 +1,12 @@
-package com.example.Flight;
+package com.example.flight;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 public class FlightApplication {
 
-	public static void main(String[] args) {
+	public static void main(String... args) {
```

Comment:
Why this change?

Reply:
Actually, there isn't a particular reason for that change, I just like the three dots